src/metabase/models/cloud_migration.clj

```diff
@@ -98,17 +106,20 @@
   (let [table-name (t2/table-name model)]
     (when (and (read-only-mode)
                (read-only-mode-inclusions table-name)
-               (not (read-only-mode-exceptions table-name))
-               (not *ignore-read-only-mode*))
+               (not (read-only-mode-exceptions table-name)))
       (throw (ex-info (tru "Metabase is in read-only-mode mode!")
                       {:status-code 403}))))
   resolved-query)
 
 
 ;; Helpers
 
-(defn- migration-action-url [external-id path]
-  (str (migration-store-url) "/" external-id path))
+(defn migration-url
+  "Store API URL for migrations."
+  ([]
+   (str (store-api-url) "/api/v2/migration"))
```

Comment:
nit: This arity is only used once, what do you think about inlining the 1-arity version of `migration-url`?

Reply:
It's used twice: once in itself (`migration-url`) to build the base part of the url and then add the path, and once in `get-store-migration`. Inlining would be a bit of a semantic loss and more verbose here I think.

Reply:
Makes sense, thanks. Also, really glad to see the versioned endpoint. 🙏